### ISSUE TITLE
limit Websocket rate?

### ISSUE THREAD

**Issue by MichalPitr** [ftexchange/ftx#55]
Hi, I am trying to subscribe to multiple futures markets orderbooks (30~), but as FTX sends live orderbook data, my cpu isn't catching up and the data is delayed. Is there some way to limit the websocket data is received? e.g. every 100ms. 
I am using Python.

Many thanks.

**Comment by arbitrage-technology** [comment]
guy, this,is basic producer consumer pattern. google it.

**Comment by reasn** [comment]
No there isn't; you can reduce the number of subscriptions per socket and drop messages after they arrive.

**Comment by reasn** [comment]
> guy, this,is basic producer consumer pattern. google it.

You're right it is, but that doesn't mean there shouldn't be a parameter to tell FTX to send less data.
